pkg/controller/unifiedpushserver/backups.go

```diff
@@ -0,0 +1,130 @@
+package unifiedpushserver
+
+import (
+	"fmt"
+
+	pushv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/push/v1alpha1"
+	batchv1 "k8s.io/api/batch/v1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func backups(ups *pushv1alpha1.UnifiedPushServer) ([]batchv1beta1.CronJob, error) {
+	cronjobs := []batchv1beta1.CronJob{}
+	for _, upsBackup := range ups.Spec.Backups {
+		cronJobLabels := labels(ups, "backup")
+		for k, v := range upsBackup.Labels {
+			cronJobLabels[k] = v
```

Comment:
Why it is required here?

Reply:
I think it's mostly because it's already populated with some values (line 16), and this is just adding any optional other ones that have been specified in the CR for this particular backup

Reply:
I think it is not required at all.